python/ray/train/torch/config.py

```diff
@@ -155,9 +140,6 @@ def on_start(self, worker_group: WorkerGroup, backend_config: TorchConfig):
             else:
                 backend = backend_config.backend
 
-            if backend == "nccl":
```

Comment:
Previously seems that we'll pipe the `NCCL_SOCKET_IFNAME` from the driver to the workers. Now the users have to either set the env variable via `ray.init(runtime_env)` or bake it into the docker. Shall we document it some where e.g. (https://docs.ray.io/en/master/train/user-guides/using-gpus.html)?

Reply:
Oh ic, you are right. It only sets the default values if the env var is empty. Thank you!